circle.yml

```diff
@@ -1,11 +1,11 @@
 machine:
   node:
-    version: 0.10.33
-  pre:
-    - curl https://install.meteor.com | /bin/sh
+    version: 6.0.0
```

Comment:
Do to:
```/home/ubuntu/testing/src/space-testing.js:121
    for (let [index, struct] of ObjectEntries(actual)) {
                             ^^

SyntaxError: Unexpected identifier
```


Can't figure out why this works above `6.0.0`, tried `4.0.0`, `5.0.0` and one from `0.10.*` - no luck. This should work on those versions. https://github.com/babel/babel/pull/5041